2.0.0/arm64/entrypoint.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -x
+set -euo pipefail
+IFS=$'\n\t'
+
+# Add openhab user & handle possible device groups for different host systems
+# Container base image puts dialout on group id 20, uucp on id 10
+# GPIO Group for RPI access
+NEW_USER_ID=${USER_ID:-9001}
+echo "Starting with UID : $NEW_USER_ID"
+adduser -u $NEW_USER_ID --disabled-password --gecos '' --home ${APPDIR} openhab &&\
```

Comment:
if it is really needed to define the uid using environment variables, it would be better to check on existence of the specific uid and only add it if not already there.